Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContainer.js

```diff
@@ -1,2 +1,27 @@
 import React from 'react';
 import './Header.css';
+
+import HeaderTitle from './HeaderTitle';
+import ImageThumbnail from './ImageThumbnail';
+import HeaderContent from './HeaderContent';
+
+const HeaderContainer = () => {
+    const headerTitleProps = {
+        h1Text: 'Lambda School',
+        spanText: '@LambdaSchool • 28 May'
+    }
+
+    return (
+        <div className="header-container">
+            <ImageThumbnail src="./imgs/lambda_logo.png" />
+            <div className="header-content">
+                <HeaderTitle headerTitleProps={headerTitleProps} />
+                <HeaderContent text="Let's learn React by building simple interfaces with components. 
```

Comment:
Though this is a clever assignment for the 'text' value: best practices is simply to use the component <HeaderContent /> ... and put the text inside that component, so you're not drilling for props on every component.